lib/request/encoder.ex

```diff
@@ -83,15 +83,15 @@ defmodule Burox.Request.Encoder do
   @spec validate_credentials(
     {String.t(), String.t(), String.t()}
   ) :: {String.t(), String.t(), String.t()}
+  defp validate_credentials({nil, nil, _version}), do: raise Burox.Error,
+    message: "Deben configurarse las credenciales del buró de crédito"
```

Comment:
Un posible _refactor_ podría ser eliminar los demás _pattern match_ que validan los `nil` y dejar solo uno con una _guard clause_ tipo:

```suggestion
  defp validate_credentials({user, password, _version}) when is_nil(user) or is_nil(password), 
    do: raise Burox.Error, message: "Deben configurarse las credenciales del buró de crédito"
```

Reply:
Si, tienes razón, se aplicó en este commit cc3fa47